### PR TITLE
Add contributor and creator to compact title

### PR DESCRIPTION
**Pull request by literarymachine**
Contributors and creators are added when viewing an entity in 'compact' mode, e.g. in a result listing.

### CODE REVIEW

**Comment by jschnasse** [comment]
on test

**Comment by jschnasse** [comment]
Still todo 
- make a single column for "zuletz hinzugefügtes Label". 
- show icons for accessScheme and publishScheme 
what else?

**Comment by literarymachine** [comment]
> show icons for accessScheme and publishScheme what else?

Any recommendation on where to put these, and which to use?

**Comment by jschnasse** [comment]
accessScheme:private <i class="fa fa-lock" color="red"></i>
accessScheme:subscriber <i class="fa fa-lock" color="orange"></i>
accessScheme:restricted <i class="fa fa-lock" color="orange"></i>
accessScheme:remote <i class="fa fa-lock" color="yellow"></i>
accessSchem:public <i class="fa fa-unlock" color="green"></i>

publishScheme:public <i class="fa fa-circle-o" color="green"></i>
publishScheme:private <i class="fa fa-circle-o" color="red"></i>

Maybe to title column?

**Comment by jschnasse** [comment]
recent comment updated!

**Comment by literarymachine** [comment]
> Maybe to title column?

If we add these here, they won't be immediately visible if we incorporate the drop-down functionality deployed at https://edoweb-test.hbz-nrw.de/resource

**Comment by jschnasse** [comment]
Based on the given requirements I see two options: 
1. Put the two icons in one row with title and contributor.
2. add another column
We should try it out.

**Comment by jschnasse** [comment]
on test

File: edoweb/edoweb.module [hunk 1]
```diff
@@ -2483,14 +2504,46 @@ function _edoweb_format_date($timestamp) {
...
 function _edoweb_format_subject($remote_id) {
   $curie = _edoweb_compact_uri($remote_id);
   return strtoupper(substr($curie, 0, strpos($curie, ':')));
 }
 
-function _edoweb_link_lastmodified($remote_id) {
-  if (_is_edoweb_entity($remote_id)) {
-    $curie = _edoweb_compact_uri($remote_id);
+function _edoweb_link_lastmodified($entity) {
+  if (('journal' == $entity->bundle()) && _is_edoweb_entity($entity)) {
```
Comment (jschnasse) [inline edoweb/edoweb.module]: I'd prefer `if ( _is_edoweb_entity($entity))`

Comment (literarymachine) [inline edoweb/edoweb.module]: I already had this for all resources types and then saw somewhere the original requirement was to only display this for journals. Will change this!

**Pull request merged**